addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/StationHandler.java

```diff
@@ -108,6 +109,17 @@ public void dispose() {
         super.dispose();
     }
 
+    @Override
+    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
+        logger.debug("Bridge Status updated to {} for device: {}", bridgeStatusInfo.getStatus().toString(),
```

Comment:
You don't have to call toString it will be called automatically, but only when the logging-level actually applies.